docs/rules/jsx-no-literals.md

```diff
@@ -1,11 +1,12 @@
-# Prevent usage of unwrapped JSX strings (react/jsx-no-literals)
+# Prevent usage of string literals in JSX (react/jsx-no-literals)
 
-In JSX when using a literal string you can wrap it in a JSX container `{'TEXT'}`.
-This rules requires that you wrap all literal strings.
-Prevents any odd artifacts of highlighters if your unwrapped string contains an enclosing character like `'` in contractions and enforces consistency.
+There are a couple of scenarios where you want to avoid string literals in JSX.  Either to enforce consistency and reducing strange behaviour, or for enforcing that literals aren't kept in JSX so they can be translated.
 
 ## Rule Details
 
+In JSX when using a literal string you can wrap it in a JSX container `{'TEXT'}`. This rules by default requires that you wrap all literal strings.
+Prevents any odd artifacts of highlighters if your unwrapped string contains an enclosing character like `'` in contractions and enforces consistency.
```

Comment:
this is the original text, essentially, with some modifications because of the different structure. If what you're saying is true, then this rule has/had no purpose.